lib/workers/repository/updates/flatten.ts

```diff
@@ -216,8 +217,14 @@ export async function flattenUpdates(
       update.semanticCommits = semanticCommits;
     }
   }
-  return updates
-    .filter((update) => update.enabled)
+  const filteredUpdates = updates
+    .filter((update) => update.enabled !== false)
```

Comment:
Is that attribute optional, if yes it changes the behavior

Reply:
see previous comments, it seems `enabled` is never `null` or `undefined`